### PR TITLE
Improve implement for pekko-connectors-text

### PR DESCRIPTION
**Pull request by laglangyue**
More direct and simple implementation

### CODE REVIEW

File: text/src/main/scala/org/apache/pekko/stream/connectors/text/javadsl/TextFlow.scala [hunk 1]
```diff
@@ -30,10 +30,7 @@ object TextFlow {
    * Decodes a stream of bytes into a stream of characters, using the supplied charset.
    */
   def decoding(incoming: Charset): Flow[ByteString, String, NotUsed] =
-    pekko.stream.scaladsl
-      .Flow[ByteString]
-      .via(new CharsetDecodingFlow(incoming))
-      .asJava
+    pekko.stream.javadsl.Flow.fromGraph(new CharsetDecodingFlow(incoming))
```
Comment (Roiocam) [inline text/src/main/scala/org/apache/pekko/stream/connectors/text/javadsl/TextFlow.scala:33]: `pekko.stream.javadsl.Flow.fromGraph` will be use the same `scaladsl` in the end, this change may break the consistency of code style, you may want to use `pekko.stream.scaladsl.Flow.fromGraph(....).asJava`

https://github.com/apache/pekko/blob/098ab4a298bc5805b9eb851314de88fd0bef4f39/stream/src/main/scala/org/apache/pekko/stream/javadsl/Flow.scala#L78-L82

File: text/src/main/scala/org/apache/pekko/stream/connectors/text/scaladsl/TextFlow.scala [hunk 2]
```diff
@@ -44,7 +43,6 @@ object TextFlow {
    * Translates a stream of bytes from one character encoding into another.
    */
   def transcoding(incoming: Charset, outgoing: Charset): Flow[ByteString, ByteString, NotUsed] =
-    Flow[ByteString]
-      .via(new CharsetTranscodingFlow(incoming, outgoing))
+    Flow.fromGraph(new CharsetTranscodingFlow(incoming, outgoing))
```
Comment (Roiocam) [inline text/src/main/scala/org/apache/pekko/stream/connectors/text/scaladsl/TextFlow.scala:46]: Would you like to take a look? which better?

@He-Pin

File: text/src/main/scala/org/apache/pekko/stream/connectors/text/impl/CharsetDecodingFlow.scala [hunk 3]
```diff
@@ -14,7 +14,6 @@
 package org.apache.pekko.stream.connectors.text.impl
 
 import java.nio.charset.Charset
-
```
Comment (He-Pin) [inline text/src/main/scala/org/apache/pekko/stream/connectors/text/impl/CharsetDecodingFlow.scala:17]: Better we keep this blank line

File: text/src/main/scala/org/apache/pekko/stream/connectors/text/javadsl/TextFlow.scala [hunk 4]
```diff
@@ -45,9 +42,6 @@ object TextFlow {
    * Translates a stream of bytes from one character encoding into another.
    */
   def transcoding(incoming: Charset, outgoing: Charset): Flow[ByteString, ByteString, NotUsed] =
-    pekko.stream.scaladsl
-      .Flow[ByteString]
-      .via(new CharsetTranscodingFlow(incoming, outgoing))
-      .asJava
```
Comment (He-Pin) [inline text/src/main/scala/org/apache/pekko/stream/connectors/text/javadsl/TextFlow.scala:51]: I have to say, they are same code after all

File: text/src/main/scala/org/apache/pekko/stream/connectors/text/impl/CharsetDecodingFlow.scala (hunk 3 above)
Comment (laglangyue) [reply text/src/main/scala/org/apache/pekko/stream/connectors/text/impl/CharsetDecodingFlow.scala]: keep it

File: text/src/main/scala/org/apache/pekko/stream/connectors/text/javadsl/TextFlow.scala (hunk 1 above)
Comment (laglangyue) [reply text/src/main/scala/org/apache/pekko/stream/connectors/text/javadsl/TextFlow.scala]: update by using scaladsl.TextFlow

File: text/src/main/scala/org/apache/pekko/stream/connectors/text/javadsl/TextFlow.scala (hunk 4 above)
Comment (laglangyue) [reply text/src/main/scala/org/apache/pekko/stream/connectors/text/javadsl/TextFlow.scala:51]: thanks, I update it

**Comment by pjfanning** [comment]
@Roiocam @He-Pin what do you think about these changes? I'm hoping to do a 1.1.0-M1 release soon so I'm looking to get PRs closed.

**Review by He-Pin — APPROVED** [review]
Lgtm，java use javadsl after all

**Review by raboof — APPROVED** [review]

**Pull request merged**